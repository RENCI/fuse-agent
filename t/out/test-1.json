{
  "configuredServices": {
    "providers": [
      {
        "URL": "http://fuse-provider-upload:8000",
        "id": "fuse-provider-upload",
        "title": "Custom data"
      }
    ],
    "tools": [
      {
        "URL": "http://fuse-tool-pca:8000",
        "id": "fuse-tool-pca",
        "title": "Principal Components Analyzer"
      }
    ]
  }
}
