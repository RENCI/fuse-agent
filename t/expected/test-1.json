{
  "configuredServices": {
    "providers": [
      {
        "URL": "http://fuse-provider-upload:8000",
        "container-network": "fuse",
        "id": "fuse-provider-upload",
        "title": "Custom data"
      }
    ],
    "tools": [
      {
        "URL": "http://fuse-tool-pca:8000",
        "container-network": "fuse",
        "id": "fuse-tool-pca",
        "title": "Principal Components Analyzer"
      }
    ]
  }
}
